pkg/apisix/cluster.go

```diff
@@ -136,6 +138,9 @@ func newCluster(o *ClusterOptions) (Cluster, error) {
 
 	go c.syncCache()
 
+	ticker := time.NewTicker(o.SyncInterval)
```

Comment:
Since this ticker is used by `syncSchema`, just created it inside it.

Reply:
Also, ticker should be closed explicitly.

Reply:
here should i put the code that closing the ticker?